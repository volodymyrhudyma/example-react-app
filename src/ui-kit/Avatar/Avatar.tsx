import React from 'react';
import { Wrapper, StyledImage } from './styled/Avatar.styled';

interface IAvatarProps {
  src: string;
  width: number;
  height: number;
}

const Avatar = (props: IAvatarProps) => (
  <Wrapper>
    <StyledImage {...props} />
  </Wrapper>
);

Avatar.defaultProps = {
  width: 60,
  height: 60,
};

export default Avatar;
