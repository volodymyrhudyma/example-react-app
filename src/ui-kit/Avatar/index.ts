import Component from './Avatar';

export default Component;
