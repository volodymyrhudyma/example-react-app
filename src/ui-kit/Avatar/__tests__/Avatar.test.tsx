import React from 'react';
import { shallow } from 'enzyme';
import Avatar from '../Avatar';

describe('Avatar component', () => {
  it('should render correctly', () => {
    const component = shallow(<Avatar src="avatar" />);
    expect(component).toMatchSnapshot();
  });
});
