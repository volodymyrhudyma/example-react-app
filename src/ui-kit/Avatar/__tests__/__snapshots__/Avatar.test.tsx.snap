// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component should render correctly 1`] = `
<styled.div>
  <styled.img
    height={60}
    src="avatar"
    width={60}
  />
</styled.div>
`;
